tests/test_integration.py

```diff
@@ -18,6 +20,10 @@ def _setup_source(setup_name, tmp_dir: Path):
     return setup_path, pypkg_dir
 
 
+@pytest.mark.skipif(
+    platform.system() == "Windows",
+    reason="Windows env variable are not easy to manage.",
+)
```

Comment:
Robust cythonization is required on Windows as well...
See with @ncian-gpsw or @sdeker-gpsw if we can keep this test ?

Reply:
we can make another exclusive test for windows with the following commands:
`virtualenv.run("$env:CYTHONIZE=1")`
`virtualenv.run(f"pip install -e {pypkg_dir}")
`virtualenv.run('$env:CYTHONIZE=""')`